### PR TITLE
live_snapshot_basic: fix wrong format specifiers

### PR DESCRIPTION
**Pull request by PandaWei**
replace '%' with '%s'  and remove unused variable.

Signed-off-by: Wei Jiangang <weijg.fnst@cn.fujitsu.com>

### CODE REVIEW

File: qemu/tests/live_snapshot_basic.py [hunk 1]
```diff
@@ -37,7 +37,6 @@ def create_image(self):
         """
         image_name = self.params.get("image_name")
         self.params['image_name_snapshot'] = image_name + "-snap"
-        image_name_snapshot = self.params['image_name_snapshot']
```
Comment (CongLi) [inline qemu/tests/live_snapshot_basic.py]: It's used in next line.

Comment (PandaWei) [inline qemu/tests/live_snapshot_basic.py]: Not really ...
The following line will use elf.params['image_name_snapshot'], not image_name_snapshot.
And I have tested it with removing the line "image_name_snapshot = self.params['image_name_snapshot']".

The case works well.

Comment (xutian) [inline qemu/tests/live_snapshot_basic.py]: @CongLi  checked your original code, not found where used image_name_snapshot

Comment (CongLi) [inline qemu/tests/live_snapshot_basic.py]: @PandaWei @xutian 

Yes, you are right. I have thought the previous line.

Acked-by: Cong Li <coli@redhat.com>

Thanks.

**Comment by xutian** [comment]
thanks @PandaWei , merged

**Pull request merged**